syntax = "proto3";

package v1;

message GreetingRequest {
  string name = 1;
}

message GreetingResponse {
  string message = 1;
}


service GreetingService {
  rpc SayHello(GreetingRequest) returns (GreetingResponse);
}
